packages/orchestration/src/orchestration-stream-response.ts

```diff
@@ -41,6 +50,38 @@ export class OrchestrationStreamResponse<T> {
     this._finishReasons = finishReasons;
   }
 
+  public getToolCalls(choiceIndex = 0): MessageToolCalls | undefined {
```

Comment:
[pp] I saw the other public method also does not have jsdoc. I guess good to have?